### PR TITLE
Update custom-theme.md with Workable example

### PR DESCRIPTION
**Pull request by albertcausing**
Workable Example for custom theme.

### CODE REVIEW

File: extend/custom-theme.md [hunk 1]
```diff
@@ -104,3 +104,109 @@ app.Settings().SetTheme(&myTheme{})
 
 With these changes you can apply your own style, to make small tweaks or
 provide a completely custom looking application!
+
+### Workable Example
```
Comment (Jacalz) [inline extend/custom-theme.md:108]: I looked at the other tutorials and it seems to me like they don’t add a new section just for the code. We probably should stay consistent.
```suggestion
In the end, the resulting code could look something like this:
```

File: extend/custom-theme.md [hunk 2]
```diff
@@ -104,3 +104,109 @@ app.Settings().SetTheme(&myTheme{})
...
+```
```
Comment (Jacalz) [inline extend/custom-theme.md:109]: Let’s add some syntax highlighting to the code. It will be basically unreadable without.
```suggestion
```go
```

**Review by Jacalz — CHANGES_REQUESTED** [review]
Thanks for the example. I have a few suggestions for how it can be improved.

**Comment by andydotxyz** [comment]
There is a lot of duplicated code - can we make the example a lot shorter as this is primarily documentation?

**Review by Jacalz — CHANGES_REQUESTED** [review]
I agree with Andy. We should ideally try to make the code example a bit shorter.

**Comment by andydotxyz** [comment]
Feels pretty stale, please do consider bringing a shortened version to the docs.fyne.io repo.

**Pull request closed without merging**